src/Schema/Exception/SequenceDoesNotExist.php

```diff
@@ -17,7 +17,7 @@ public static function new(string $sequenceName): self
     {
         return new self(
             sprintf('There exists no sequence with the name "%s".', $sequenceName),
-            self::SEQUENCE_DOENST_EXIST
+            self::SEQUENCE_DOESNT_EXIST
```

Comment:
This exception has a unique class by which it can be identified both in code and in logs. Why does it need a numeric error code?

Reply:
No worries. I thought this was a no-brainer until I realized that this was introduced in 2.x, not 3.x.
I wouldn't worry that much about the BC break in 3.x though, considering the amount of breaking changes, this one is minor.

Do you want to close this anyway?

Reply:
Yes, I’d close it.

Reply:
Alright!